pkg/devfile/parser/parse.go

```diff
@@ -374,6 +378,13 @@ func parseFromURI(importReference v1.ImportReference, curDevfileCtx devfileCtx.D
 	if !absoluteURL && curDevfileCtx.GetAbsPath() != "" {
 		newUri = path.Join(path.Dir(curDevfileCtx.GetAbsPath()), uri)
 		d.Ctx = devfileCtx.NewDevfileCtx(newUri)
+		if util.ValidateFile(newUri) == nil {
```

Comment:
error handling. user should know if the parent folder (not exist/not readable/not writable ext. )